src/style.css

```diff
@@ -1,3 +1,92 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+:root {
+  --blue-color: #00ACDC;
+
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  margin: auto 0;
+}
+
+
+ul {
+  padding-inline-start: 0;
+}
+
+.header {
+  width: auto;
+  display: flex;
+  background-color: #FFF;
+  justify-content: space-between;
+  text-align: center;
+  padding: 0 50px;
+}
+
+.nav_link {
+  color: #000;
+  display: flex;
+  text-transform: uppercase;
+  font-weight: 500px;
+  text-decoration: none;
+  height: 60px;
+  text-align: center;
+  width: auto;
+  align-items: center;
+  margin-left: 20px;
```

Comment:
You should avoid margin collapsing, when a child margins go through a parent and push off another parent
[useful_link](https://www.smashingmagazine.com/2019/07/margins-in-css/#parent-and-first-or-last-child-element)
So, if you want one element to be push off from another element use margins at THESE elements, not their children

Also, first item shouldn't have margin